### PR TITLE
Pull resources from submodules instead of copying from submodules

### PR DESCRIPTION
**Pull request by mistermoe**
TODO:
* [x] remove gradle tasks to copy stuff into resources
* [x] delete schemas from `src/main/resources`
* [x] potentially adjust github action to pull submodule

### CODE REVIEW

File: protocol/build.gradle.kts [hunk 1]
```diff
@@ -33,15 +33,20 @@ dependencies {
...
+    val resourceDirs = listOf(
+      "../tbdex/hosted/test-vectors/protocol/vectors",
+    )
+    resources.setSrcDirs(resourceDirs)
   }
 
-  register("syncTestVectors", Sync::class) {
-    from("../tbdex/hosted/test-vectors/protocol/vectors")
-    into("./src/test/resources/test-vectors")
+  main {
```
Comment (jiyoontbd) [inline protocol/build.gradle.kts:44]: not really sure the diff between this vs using `val main by getting {..` but this one works! 🤷

**Comment by jiyoontbd** [comment]
@mistermoe 
> potentially adjust github action to pull submodule
so we need to add some actions in `ci.yaml` to pull in submodule changes?

**Comment by diehuxx** [comment]
@jiyoontbd 

When checking out code in a github action you have to opt in to checking out the submodule as well. See this action in tbdex-js with `submodules: true`: https://github.com/TBD54566975/tbdex-js/blob/main/.github/workflows/integrity-check.yml#L43-L44

**Comment by mistermoe** [comment]
This is good to go yeah?

**Comment by jiyoontbd** [comment]
@mistermoe as far as i can tell, yea it is ready to go :D but needs stamps

**Comment by mistermoe** [comment]
1 stamp from u maybe?

**Review by jiyoontbd — APPROVED** [review]
😲 wow i can approve this pr

**Comment by mistermoe** [comment]
going to go ahead and merge this. @jiyoontbd and i are aware of the changes made bc we made them together and @diehuxx is in _the know_ about submodules and such

**Pull request merged**